lib/core/player/Player.c

```diff
@@ -0,0 +1,18 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "Player.h"
+
+Player* buildPlayer(char* name, unsigned int tokens) {
+    Player* player = malloc(sizeof(Player));
+    if(player) {
```

Comment:
General c question. Why do you need to check if player?

Reply:
If we fail to reserve memory, malloc will return NULL. So we need to check that we aren't working with a null pointer. That check if equivalent to:
```c
if(player != NULL) {
    ....
}
```